📝 Updates to checklists from C-PAC tool certification walkthrough (#8)
nmind/coding-standards-certification: checklists/bronze.md

### Documentation

> - [ ] README provides link to documentation

Reviewer: To the question in the PR itself, whether or not we require the software to be open source, a README is a very "GitHub" concept, so I may suggest that we instead say something like the following:
```suggestion
- [ ] Landing page (e.g., GitHub README, website) provides a link to documentation
```

> - [ ] Documentation is up-to-date

Reviewer: Perhaps more specifically: 
```suggestion
- [ ] Documentation corresponds to latest version of software
```

> - [ ] Brief description of what program does is listed

Reviewer: I think we want more specificity here; does it matter where it is? I think we mean that it's easy to find, so something like the following may be more precise:
```suggestion
- [ ] Brief description of what program does can be found on the landing page
```

> - [ ] Typical intended usage is described

Reviewer: Again, I'd think of this in the context of the landing page. Maybe what we want is to nest criteria here? Some of these should certainly be accessible from the high-level landing page, and others in the more dense documentation. If we make sub-lists for each, it could help reviewers and make sure that the pieces are easy to find for consumers.

- [ ] An example of its usage is shown
- [ ] User installation instructions available
- [ ] Dependencies listed (external and within-langugage requirements)

> - [ ] Description of required input parameters with reasonable description of inputs (i.e. "Nifti of brain mask in MNI" vs. "An image file")

Reviewer: ```suggestion
- [ ] Description of required input parameters for deliberately user-facing functions with reasonable description of inputs (i.e. "Nifti of brain mask in MNI" vs. "An image file")
```

> - [ ] Description of output(s)

Reviewer: ```suggestion
- [ ] Description of the output(s) for deliberately user-facing functions
```

> - [ ] Document functions intended to be used by users (public function docstring / help coverage ≥ 10%)

Reviewer: I think we'd bump this one up in the list above the preceding 2
